examples/src/TimeLabel.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { useAudioPosition } from "react-use-audio-player"
+
+const formatTime = (seconds: number) => {
+    const floored = Math.floor(seconds)
+    let from = 14
+    let length = 5
+    // Display hours only if necessary.
+    if (floored >= 3600) {
+        from = 11
+        length = 8
+    }
+
+    return new Date(floored * 1000).toISOString().substr(from, length)
+}
+
+export const TimeLabel = () => {
+    const { duration, position } = useAudioPosition({ highRefreshRate: true })
+    const elapsed = typeof position === 'number' ? position : 0
```

Comment:
Still had this annoying bug so I had to add the check.

Reply:
damn, okay i know what i have to do to make this go away for good. It ain't pretty, but better in the source than in client code. This seems like an issue that should be taken up with howler. Their own seek method behaves very strangely/counter-intuitively.